app/controllers/duplicate_logs_controller.rb

```diff
@@ -47,4 +60,11 @@ def duplicate_check_question_ids
       %w[owning_organisation_id saledate purchid age1 sex1 ecstat1 postcode_full]
     end
   end
+
+  def find_original_log_id
+    return unless @log
+
+    query_params = URI.parse(request.url).query
+    @original_log_id = query_params.present? ? CGI.parse(query_params)["original_log_id"][0]&.to_i : @log.id
```

Comment:
So when we click on `keep this log and delete duplicates` the path would be `/lettings-logs/:id_of_the_log_we_keep/duplicate-logs?original_log_id=:id_of_the_original_log`

Reply:
And sometimes they're the same, but I don't think it makes sense to keep the original log id in the path if it's deleted, especially if we allow the log to be edited afterwards

Reply:
what's the situation where we don't find it in the params and fall back on @log.id? I guess it's confusing me slightly that the original log identifier might not be kept in the query param as you have it above

Reply:
Okay, so I've updated this slightly. We can pass that ID through from the very beginning, because we know which journey we're coming from. We don't default anymore so it shouldn't interfere with your ticket as much